ai/datasets/nerrf_dataset.py

```
@@ -0,0 +1,394 @@
+            else:  # file
+                node_type = [0.0, 1.0]
+                path_hash = hash(node_data["path"]) % 1000 / 1000.0
+            
+            # Time features (hour of day, day of week)
+            timestamp = node_data["first_seen"]
+            time_features = [
+                timestamp.hour / 24.0,
+                timestamp.weekday() / 7.0
+            ]
+            
+            features = node_type + [path_hash] + time_features
```

Ruff (0.13.3)

Consider `[*node_type, path_hash, *time_features]` instead of concatenation

Replace with `[*node_type, path_hash, *time_features]`

(RUF005)